docs/02.API-REFERENCE.md

```diff
@@ -1486,6 +1486,48 @@ jerry_value_set_error_flag (jerry_value_t *value_p);
 - [jerry_value_t](#jerry_value_t)
 
 
+## jerry_get_value_without_error_flag
+
+**Summary**
+
+If the input value is a error value, then return its reference value.
+Otherwise return a new reference to itself.
+
+*Note*: Returned value must be freed with [jerry_release_value](#jerry_release_value) 
+when it is no longer needed.
+
+**Prototype**
+
+```c
+jerry_value_t 
+jerry_get_value_without_error_flag (jerry_value_t value)
+```
+
+- `value` - api value
+
+**Example**
+
+```c
+{
+  jerry_value_t value;
+  ... // create or acquire value
+  
+  jerry_value_set_error_flag (&value);
+  
+  jerry_value_t real_value = jerry_get_value_without_error_flag (value);
+  ... // process the real_value. Different from `jerry_value_clear_error_flag`, 
+      // the error `value` will not be automatically released after calling 
+      // `jerry_get_value_without_error_flag`.
+
+  jerry_release_value (value);
+  jerry_release_value (real_value);
+}
+```
+
+**See also**
+
+- [jerry_value_clear_error_flag](#jerry_value_clear_error_flag)
```

Comment:
Add acquire_value as reference. Actually this function is closer to acquire_value than clear_error_flag.